src/views/Predictions/Positions.tsx

```diff
@@ -22,19 +23,23 @@ const StyledSwiper = styled.div`
     width: 320px;
   }
 `
-const Positions: React.FC = () => {
+const Positions: React.FC<{ view?: string }> = ({ view }) => {
   const { setSwiper } = useSwiper()
   const rounds = useGetSortedRounds()
-  const initialIndex = Math.floor(rounds.length / 2)
+  const currentEpoch = useGetCurrentEpoch()
+  const liveEpoch = currentEpoch > 0 ? currentEpoch - 1 : currentEpoch
```

Comment:
Can we avoid calling this "live" - it is already confusing enough that current is actually "next" on the UI.

Reply:
Aha yea I wasn't sure how best to name this. I'll go with `previousRound` for now given it's `currentRound` minus 1.

Would that match the naming convention you've used elsewhere?